### DISCUSSION TITLE
SMTP not working

### DISCUSSION THREAD

**Discussion by SKJoy** [mediacms-io/mediacms#1123] [Q&A]
**Describe the issue**
SMTP settings been set as per instruction but contact form notification is not being sent to admin emails.

**To Reproduce**
Steps to reproduce the issue:
1. Set SMTP settings in settings.py (as well in local_settings.py)
2. Restart MediCMS server
3. Go to MediaCMS Contact form and submit a message
4. Notification never received to the admin emails

**Expected behavior**
Contact form message notification should arrive at admin emails.

**Environment (please complete the following information):**
 - OS: Ubuntu Linux
 - Installation method: Single server install

**Additional context**
## SMTP settings
EMAIL_HOST = "mail.smtp_server.com"
EMAIL_HOST_USER = "mediacms.test@smtp_server.com"
EMAIL_HOST_PASSWORD = "CORRECT_PASSWORD"
EMAIL_PORT = 587
EMAIL_USE_TLS = True
DEFAULT_FROM_EMAIL = "mediacms.test@smtp_server.com"
SERVER_EMAIL = DEFAULT_FROM_EMAIL
ADMIN_EMAIL_LIST = ["admin_1@domain.com", "admin_2@domain.com"]

**Test information**
- SMTP settings were tested with WebMail & Mail clients and been found 100% working.
- Tested with manage.py SHELL and the test succeeded, we received test emails

**Comment by SKJoy** [comment]
The SMTP configuration seemed to work after a server REBOOT! Is there a simpler way to reflect the configuration without a server reboot?

**Comment by Utility9298** [comment]
I also did recognize that the SMTP configuration needs a server reboot... 
(Running single server install on Ubuntu 22.04)

I also can't get it running without smtp username and username passowrd (m365 mail connector does not need username/passowrd because it gets validated via IP Adress....)

**Reply by mgogoulos** [reply]
i guess it needs some development in order for this to be supported...

**Reply by Utility9298** [reply]
@mgogoulos 

 I got it running after some testing using following settings with the m365 mail connector. It does need port 25 and use tls:

`# Mail Settings
DEFAULT_FROM_EMAIL = 'mediacms@companydomain.at'
EMAIL_HOST_PASSWORD = ''
EMAIL_HOST_USER = ''
SERVER_EMAIL = DEFAULT_FROM_EMAIL
EMAIL_HOST = 'company.at.mail.protection.outlook.com'
EMAIL_PORT = 25
EMAIL_USE_TLS = True
ADMIN_EMAIL_LIST = ["mail@company.at"]`

**Comment by mgogoulos** [comment]
it's actually the celery_worker that needs to be restarted in order for new smtp options to take effect, so this is what the reboot has accomplished. 

ideally any setting like this will be moved to the database level, so no file editing and no restarts will be necessary.
